src/commonMain/kotlin/fr/acinq/lightning/io/Peer.kt

```diff
@@ -281,7 +279,7 @@ class Peer(
     fun disconnect() {
         if (this::socket.isInitialized) socket.close()
         _connectionState.value = Connection.CLOSED(null)
-        output.close()
+        // TODO in exception handler? output.close()
```

Comment:
I'm not sure where to do this, I believe we should probably copy the clean up code from ElectrumClient?